stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/StorageHelper.kt

```diff
@@ -134,8 +133,7 @@ public class StorageHelper {
         return attachments
     }
 
-    private fun getFileName(attachmentMetaData: AttachmentMetaData): String =
-        "STREAM_${dateFormat.format(Date().time)}_${attachmentMetaData.getTitleWithExtension()}"
```

Comment:
This was here to avoid problems when multiple files with the same name are present, see the commit and its description here https://github.com/GetStream/stream-chat-android/commit/0c553c82a9c717b6ee27271bdc2f8a1ae672ae1a. Won't we have problems with that with the changes introduced here?

Reply:
(That commit only introduced the milliseconds part specifically, but apparently even that was necessary?)

Reply:
🤔  I didn't dig much in it. We can add the mili seconds in the end, anyways

Reply:
As far as I remember, the milliseconds part was added because every GIF sent from the keyboard has the same name. If you were sending them fast enough (or there was another case when you were sending multiple files with the same name) cached files were overridden and you were sending n * the same image instead of correct ones.

Reply:
I kept the prefix, but now it is smaller and we can show items that are more than one line long.

Reply:
Edit: The prefix is added to the file, but the prefix is removed when showing the title.

Reply:
At this point, couldn't we revert to the old prefix of `STREAM_yyyyMMdd_HHmmssSSS_`?

Reply:
We need something constant in the stream in the beginning and end of the prefix. We could use `STREAM_yyyyMMdd_HHmmssSSS_prefix`. What do you think?

Reply:
Nevermind... all we have to do it to replace it the first time, the original one can be kept.